packages/aws-sdk-signers/src/aws_sdk_signers/__init__.py

```diff
@@ -20,14 +21,14 @@
 __version__ = importlib.metadata.version("aws-sdk-signers")
 
 __all__ = (
-    "AsyncBytesReader",
-    "AsyncSigV4Signer",
-    "AsyncEventSigner",
+    "URI",
```

Comment:
??? What is this sort order? Are we using the ascii symbol table to determine ordering?

Reply:
> An isort-style sort orders items first according to their casing: SCREAMING_SNAKE_CASE names (conventionally used for global constants) come first, followed by CamelCase names (conventionally used for classes), followed by anything else. Within each category, a [natural sort](https://en.wikipedia.org/wiki/Natural_sort_order) is used to order the elements.

Source: https://docs.astral.sh/ruff/rules/unsorted-dunder-all/

Seems like `URI` got pushed to the top since it's being treated like a constant.

Reply:
This can be fixed